src/commands/components/index.ts

```diff
@@ -17,7 +17,11 @@ interface Component {
 export default class Components extends BaseCommand {
   static aliases = ['components', 'components:search', 'component:search', 'component:search'];
   static description = 'Search components you have access to';
-
+  static examples = [
+    'architect components',
+    'architect components --account=myAccount',
```

Comment:
Account names need to be lowercase - have this match our regex. Same with component.

```
architect components --account=myaccount
```

Reply:
renamed all example camelcase values to lowercase